runner/client/src/mill/client/lock/DoubleLock.java

```diff
@@ -12,22 +12,44 @@ public DoubleLock(Lock lock1, Lock lock2) throws Exception {
 
   @Override
   public Locked lock() throws Exception {
-    return new DoubleLocked(lock1.lock(), lock2.lock());
+    Locked l1 = null;
```

Comment:
Let's remove this from this PR since it's unrelated

Reply:
Should be done